### ISSUE TITLE
Can't open images with other programs

### ISSUE THREAD

**Issue by suilno** [flathub/org.xfce.ristretto#12]
Using Xubuntu 20.04, Flatpak 1.12.4 and Ristretto 0.12.2 (flatpak).

When opening an image and choosing the "open with" option under the menu bar or "edit this image" in toolbar, Ristretto doesn't list any programs installed on my system, except :

![Captura de tela_2022-01-24_11-35-43](https://user-images.githubusercontent.com/66175968/150804100-fbdfa48f-801b-4899-98e8-910a2d0a6550.png)

Already tried changing Ristretto permissions in Flatseal, but didn't get anything.

**Comment by Tamaranch** [comment]
Yes, I left that out until now. The problem is that there doesn't seem to be a proper replacement for `g_app_info_get_all()` and `g_app_info_get_all_for_type()` in flatpak context.

Maybe we should even consider that this kind of feature is not suitable/desirable in this context: I had almost greyed it out, maybe I should have…

I also couldn't find any permissions changes that allow these functions to work properly, but I may have missed something. I'll gladly review any suggestions for improvements on this, but I don't plan to spend any time on this myself in the near future.
